contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -6864,7 +6864,7 @@ exec_eval_datum(PLtsql_execstate *estate,
 				 * pass-by-reference, which will cause problems since our usage
 				 * here is not. Setting isnull to true will bypass the problem.
 				 */
-				*isnull = true;
+				// *isnull = true;
```

Comment:
not deleting it?

Reply:
good point, I'll just delete this along with the comment